masterlist.yaml

```diff
@@ -1040,8 +1040,30 @@ plugins:
     tag:
       - NoMerge
   - name: 'More Perks.esm'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/35979' ]
     tag:
       - Names
+  - name: 'More Perks for Dead Money.esm'
+    dirty:
+      - <<: *quickClean
```

Comment:
You can get rid of this`<<: *quickClean` line and the one on line 1059 below, because that merges in the `info` and `util` keys, but you override those values with the values you've specified.

Reply:
👍 I'll fix this shortly.

Reply:
Fixed.